etc/kayobe/kolla.yml

```diff
@@ -360,6 +360,8 @@ kolla_build_customizations_common:
   nova_compute_packages_append:
     - python3-libvirt
     - python3-ethtool
+  neutron_mlnx_agent_pip_packages_override:
+    - networking-mlnx@git+https://github.com/stackhpc/networking-mlnx@stackhpc/{{ openstack_release }}
```

Comment:
@jovial Is this correct for Caracal?

Reply:
Yes, we still need our fork as the changes haven't landed upstream: https://github.com/stackhpc/networking-mlnx/tree/stackhpc/2024.1